src/google/protobuf/compiler/objectivec/BUILD.bazel

```diff
@@ -6,6 +6,15 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
 load("//build_defs:cpp_opts.bzl", "COPTS")
 
+cc_library(
```

Comment:
This only exposes the header for folks to include it, how do they link against the code?

Reply:
Good point, lemme refactor this with public/private headers

Reply:
I refactored these so that the "names" target also includes the definitions for linkage.  Thanks for flagging this!